python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -28,16 +28,31 @@ def predict(
         logs_dir: Optional[str] = None,
         logging_frequency: int = 25,
     ):
+
```

Comment:
there is no doc for class RankingModel

cf https://ml4ir.readthedocs.io/en/latest/api/relevance_model.html#module-ml4ir.applications.ranking.model.ranking_model.RankingModel